WFP-VAM/prism-app
# Fix EWS-1294 date missing in the CSV download; COUNTRY=cambodia

### Description

This fixes #1049 .

Fixes a bug where only the time would appear in the CSV for Cambodia - EWS-1294 layer

## How to test the feature:

- select Cambodia -> Flood -> Early Warning
- select array to display chart on the map
- use download CSV button and verify the date show both date and time

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
